app/models/vhost.js

```diff
@@ -2,20 +2,24 @@ import DS from 'ember-data';
 import ProvisionableMixin from '../mixins/models/provisionable';
 
 export default DS.Model.extend(ProvisionableMixin, {
-  virtualDomain: DS.attr('string'),
   externalHost: DS.attr('string'),
   privateKey: DS.attr('string'),
-  certificate: DS.attr('string'),
+  certificateBody: DS.attr('string'),
   type: DS.attr('string', {defaultValue:'http_proxy_protocol'}),
   isDefault: DS.attr('boolean'),
   internal: DS.attr('boolean', {defaultValue: false}),
 
+  certificate: DS.belongsTo('certificate', { async: true }),
   service: DS.belongsTo('service', {async:true}),
   app: DS.belongsTo('app', { async: true }),
   operations: DS.hasMany('operation', {async:true}),
 
   reloadWhileProvisioning: true,
 
+  virtualDomain: DS.attr('string'),
```

Comment:
Not sure if this is intentional here (if it is and you prefer it please disregard), but in general we keep the `DS.attr`'s at the top section of each model definition.